### PR TITLE
Setup migartions and the sql schemas

### PR DESCRIPTION
**Pull request by camilevahviraki**
### **In this pull request we did the following:**

- Build the schema.
   -  Create and run the necessary migration files.
   -  Foreign keys should be included.

### CODE REVIEW

File: db/migrate/20220810000616_add_user_ref_to_comment.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :users, null: false, foreign_key: true
```
Comment (Davidosky007) [inline db/migrate/20220810000616_add_user_ref_to_comment.rb]: * Kindly reference the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) from the sneak peek if you have any questions about table or column names or data relationships. as all foreign keys should match what we have in the `ERD` Diagram 
image

File: db/migrate/20220810000929_add_user_ref_to_post.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :users, null: false, foreign_key: true
```
Comment (Davidosky007) [inline db/migrate/20220810000929_add_user_ref_to_post.rb]: * Kindly stick to the information provided in the ERD diagram as i mentioned above to create your foreign keys

File: db/migrate/20220810001027_add_user_ref_to_like.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :users, null: false, foreign_key: true
```
Comment (Davidosky007) [inline db/migrate/20220810001027_add_user_ref_to_like.rb]: * Also here too please stick to the ERD diagram, as future milestones would depend on your databse being created properly

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @camilevahviraki 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* No linter errors 👍 
* Well written readme file 👍 
* Good looking PR description 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @camilevahviraki, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 
- Your code is clean and well documented ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by camilevahviraki** [comment]
Thank you

**Pull request merged**